tests/framework/pvc.go

```diff
@@ -492,3 +429,41 @@ func addVolumeMounts(pvc *k8sv1.PersistentVolumeClaim, volumeName string) []k8sv
 	}
 	return volumeMounts
 }
+
+// GetImageInfo returns qemu-img information about given image
+func (f *Framework) GetImageInfo(namespace *k8sv1.Namespace, pvc *k8sv1.PersistentVolumeClaim) (image.ImgInfo, error) {
+	var err error
+	var info image.ImgInfo
+	executorPod := f.startVerifierPod(namespace, pvc)
+
+	cmd := fmt.Sprintf("qemu-img info %s/disk.img --output=json", utils.DefaultPvcMountPath)
+	output, stderr, err := f.ExecShellInPod(executorPod.Name, namespace.Name, cmd)
+
+	if err != nil {
+		fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: stderr: [%s]\n", stderr)
+		return info, err
+	}
+	fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: qemu-img info output %s\n", output)
+
+	err = json.Unmarshal([]byte(output), &info)
+	if err != nil {
+		klog.Errorf("Invalid JSON:\n%s\n", string(output))
+		return info, err
+	}
+
+	return info, nil
```

Comment:
In the first two err returns info is useless, so I guess passing *image.ImgInfo arg instead would be cleaner

Reply:
Good idea